package com.savely.exception;

public class UserAlreadyAddedException extends RuntimeException {

}
